source/module_base/test/matrix3_test.cpp

```diff
@@ -1,8 +1,65 @@
 #include "../matrix3.h"
 #include "gtest/gtest.h"
+#include "gmock/gmock.h"
 #include <random>
 #include <vector>
-class matrix3_test : public testing::Test
+
+/************************************************
+ *  unit test of class Matrix3
+ ***********************************************/
+
+/**
+ * - Tested Functions:
+ *   - construct
+ *     - two ways of constructing a 3x3 matrix
+ *   - identity
+ *     - set a 3x3 matrix to identity matrix
+ *   - zero
+ *     - set all elements of a 3x3 matrix to zero
+ *   - det
+ *     - calculate the determinant of 3x3 matrix
+ *   - transpose
+ *     - do the transpose of 3x3 matrix
+ *   - inverse
+ *     - do the inverse of 3x3 matrix
+ *   - equal
```

Comment:
The name "equal" is usually considered as operator "==". 
It would be better calling "=" as "assignment".
See https://www.cplusplus.com/doc/tutorial/operators/ .